mapit_mysociety_org/urls.py

```diff
@@ -10,16 +9,20 @@
 urlpatterns = [
     url(r'^changelog$', render, {'template_name': 'changelog.html'}, 'mapit_changelog'),
     url(r'^', include('mapit.urls')),
+    url(r'^pricing/', render, {'template_name': 'pricing.html'}, 'mapit_pricing'),
+    url(r'^legal/', render, {'template_name': 'mapit/licensing.html'}, 'mapit_legal'),
+    url(r'^docs/', render, {'template_name': 'docs.html'}, 'mapit_docs'),
     url(r'^admin/', include(admin.site.urls)),
     url(r"^account/api_keys/", include("api_keys.urls")),
-    # Turn off the settings url from the account app, as we don't want it.
-    url(r"^account/settings/", page_not_found),
+    url(r"^account/subscription", include("subscriptions.urls")),
     # Override the login and signup views from the account app, so we can use
     # our versions which use an email address instead of a username.
     url(r"^account/signup/$", SignupView.as_view(), name="account_signup"),
     url(r"^account/login/$", LoginView.as_view(), name="account_login"),
     # Override the confirm_email view from the account app, so we can sign
     # people in immediately after they confirm.
     url(r"^account/confirm_email/(?P<key>\w+)/$", ConfirmEmailView.as_view(), name="account_confirm_email"),
+    url(r"^account/$", RedirectView.as_view(pattern_name='subscription', permanent=True)),
     url(r"^account/", include("account.urls")),
+    url(r'^favicon\.ico$', RedirectView.as_view(url='/static/img/favicon.ico', permanent=True)),
```

Comment:
This seems more like something that the web server should handle, could it live in the project's `httpd.conf` instead? Also, if you included the right `<link>` tag in the HTML then the browser wouldn't even need to make a request to `/favicon.ico`.